tasks/components/cards/TaskCard.tsx

```diff
@@ -62,7 +62,7 @@ export const TaskCard = ({
           </Span>
         </div>
         <div className={tw('flex flex-col gap-y-1 w-[24px]')}>
-          {assignee && <Avatar avatarUrl={assignee.avatarUrl} alt={translation.assigned} size="tiny"/>}
+          {assignee && <Avatar avatarUrl={assignee.avatarUrl ?? ''} alt={translation.assigned} size="tiny"/>}
```

Comment:
Using empty strings everywhere to circumvent proper type checking doesn't seem like a good pattern

```suggestion
          {assignee && assignee.avatarUrl && <Avatar avatarUrl={assignee.avatarUrl} alt={translation.assigned} size="tiny"/>}
```